src/components/__tests__/MovieTile.test.tsx

```diff
@@ -39,7 +48,10 @@ describe("MovieTile", () => {
 
     it("handle click on the movie tile image", () => {
         const onClick = jest.fn();
-        render(<MovieTile movieDetails={moviesData} onClick={onClick} />);
+        render(<MovieTile movieDetails={moviesData}
+                          onClick={onClick}
+                          handleDelete={() => {}}
+                          handleEdit={() => {}} />);
```

Comment:
Need increase to 80%  at least
![image](https://github.com/user-attachments/assets/923d432a-1dfa-46b6-9ebd-17eb1a5ba082)